lcb_runner/evaluation/compute_code_generation_metrics.py

```diff
@@ -16,18 +16,20 @@
 from lcb_runner.evaluation.pass_k_utils import compute_metrics_from_results
 
 
+def _temp_run(sample, generation, debug, result,metadata_list):
```

Comment:
minor but this should accept the timeout argument instead of the hardcoded value right?